firebase-app-distribution/src/main/java/com/google/firebase/appdistribution/FirebaseAppDistributionRegistrar.java

```diff
@@ -15,16 +15,24 @@
 package com.google.firebase.appdistribution;
 
 import androidx.annotation.Keep;
+import com.google.firebase.FirebaseApp;
 import com.google.firebase.components.Component;
 import com.google.firebase.components.ComponentRegistrar;
+import com.google.firebase.components.Dependency;
+import com.google.firebase.platforminfo.LibraryVersionComponent;
 import java.util.Arrays;
 import java.util.List;
 
-/** @hide */
```

Comment:
I'm curious why removed the hide annotation here? It seems like other Registrar classes have it, and @hide should prevent this class from showing up in the API docs, which I think is what we want. (https://stackoverflow.com/q/17035271)

Reply:
haha this one is on me. I wasn't sure what the hide annotation does so I asked Andrew to remove it. Sorry about that!